assets/src/edit-story/components/library/panes/shared/libraryMoveable.js

```diff
@@ -158,7 +158,7 @@ function LibraryMoveable({
     // Don't display the clone right away since otherwise it will be triggered for a click as well.
     if (
       cloneRef.current &&
-      inputEvent.timeStamp - eventTracker.current.timeStamp > 300
+      inputEvent.timeStamp - eventTracker.current.timeStamp > 50
```

Comment:
I think `50` might be a bit too low value, when I click an image "lazily" using a mouse, then sometimes nothing seems to happen, in practice, a clone is displayed on top of the image for a split second but it doesn't insert an image since the dragging logic was triggered. This should probably stay a bit higher, e.g. `100`.

Alternatively, perhaps we should add here distance tracker as well? E.g. if the mouse has moved for more than 1-2 pixels, then start the dragging.

Reply:
Ah yes, I'll rename that utility function and use that to see whether it's a drag or not. Completely forgot about that one - thanks!